node/core/pvf/Cargo.toml

```diff
@@ -20,6 +20,7 @@ gum = { package = "tracing-gum", path = "../../gum" }
 pin-project = "1.0.9"
 rand = "0.8.5"
 tempfile = "3.3.0"
+crossbeam-utils = "0.8.9"
```

Comment:
I believe adding crossbeam is fine since we already have it in the dependency tree. Alternatively, we could do the same but that will require some unsafe on our part.

Reply:
It's fine and we can remove it in August with 1.63 std: https://doc.rust-lang.org/nightly/std/thread/fn.scope.html